CHANGELOG.md

```diff
@@ -4,6 +4,265 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.14.0-RC1](https://github.com/go-gitea/gitea/releases/tag/v1.14.0) - 2021-03-09
+
+* SECURITY
+  * Add reverse proxy configuration support for remote IP address detection (#14959)
+  * Ensure validation occurs on clone addresses too (#14994)
+  * Fix several render issues highlighted during fuzzing (#14986)
```

Comment:
I would probably add this https://github.com/go-gitea/gitea/pull/15014 to security.
```suggestion
  * Fix several render issues highlighted during fuzzing (#14986)
  * Respect approved email domain list for externally validated users (#15014)
```